typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { Moment } from 'moment';
+import ReactDatetime = require('react-datetime');
```

Comment:
@AaronAsAChimp @simeg I suspect the documentation isn't up to date.. the require syntax is what typescript implemented before the ES6 syntax was ready.

The main issues here are: 
- Editors does not allow you to type `import * as ReactDatetime from 'react-datetime';` which works
- Editors allows you now to type: `import ReactDatetime from 'react-datetime';` which doesnt work.

This indicates that there is something wrong with the typings. (Also all other imports we use have no problems with the import * as- syntax, and there's a lot).
I'm sure there is a way to make the typings work without rewriting the actual code itself, unfortunately I've not had time to do further research into this.

Anyway, the problem for us is that we use tslint with `"no-var-requires": true`, but i work around that with disable-line, so it's not game breaking for me, just a heads up..